### PR TITLE
Unified docker logic

### PR DESCRIPTION
**Pull request by ethanwayda22**
# Description
Updated the docker container format for local development. This includes better documentation, a single container, and file cleanup.

# Type of change

- [ ] Bug fix (change which fixes an issue)
- [X] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please validate that you have tested your change in at least one of the following areas:

- [X] Successfully tested locally (or docker image)
- [ ] Successfully tested on a staging or sandbox server
- [ ] Successfully tested on a merchant's staging server

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have created or modified unit tests to sufficiently cover my changes.

### CODE REVIEW

File: operations/docker/docker_compose/README.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+# How to run
+
+- Set the initial variables of config.sh to match your environment and versioning
+
+```shell
+#VARIABLES
+boltApiKey=""
+boltSigningSecret=""
+boltPublishableKey=""
+ngrokUrlHTTP="https://m2.<NAME>.dev.bolt.me/"
```
Comment (daisy1754) [inline operations/docker/docker_compose/README.md]: how about script just takes host name? ie `m2.<NAME>.dev.bolt.me`

File: operations/docker/php72_integration/Dockerfile [hunk 2]
```diff
@@ -65,10 +65,12 @@ innodb_flush_log_at_trx_commit=2\n\
 sync_binlog=0\n\
 innodb_use_native_aio=0\n' >> /etc/mysql/my.cnf
 
-##########################################################################
+################ MAgento 2 Install ################
```
Comment (daisy1754) [inline operations/docker/php72_integration/Dockerfile]: Magento (small a)

File: operations/docker/docker_compose/README.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+ngrokUrlHTTPS="https://m2.<NAME.dev.bolt.me/"
+installSampleData=true
```
Comment (daisy1754) [inline operations/docker/docker_compose/README.md]: Is this still relevant?

File: operations/docker/docker_compose/README.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+- Set the initial variables of config.sh to match your environment and versioning
+
+```shell
+#VARIABLES
+boltApiKey=""
+boltSigningSecret=""
+boltPublishableKey=""
+ngrokUrlHTTP="https://m2.<NAME>.dev.bolt.me/"
+ngrokUrlHTTPS="https://m2.<NAME.dev.bolt.me/"
```
Comment (matt-thomason) [inline operations/docker/docker_compose/README.md]: `<NAME>` lost a caret here.

File: operations/docker/docker_compose/README.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+installSampleData=true
+```
+
+- Run `docker-compose up -d` to build your docker environment
+- Run **run_docker_scripts.sh** to fully deploy a docker container containing Magento2 with Bolt
+- Visit store at your set ngrokUrlHTTP and admin ngrokUrlHTTP/backend
+  - Admin credentials are username: **bolt** password: **admin123**
```
Comment (matt-thomason) [inline operations/docker/docker_compose/README.md]: Is this still true? Looks like the password is set differently below.

File: operations/docker/php72_integration/install_magento_data.sh [hunk 6]
```diff
@@ -50,30 +51,27 @@ php bin/magento setup:install \
     --admin-user="admin" \
     --use-rewrites=1 \
     --admin-use-security-key=0 \
-    --admin-password="123123q"
+    --admin-password="bolt1234"
```
Comment (matt-thomason) [inline operations/docker/php72_integration/install_magento_data.sh]: Different user/pass?

**Comment by danac-gs** [comment]
@ethanwayda22 what's the status here?

**Comment by matt-thomason** [comment]
@ethanwayda22 Ping - is this still being worked on?

**Comment by matt-thomason** [comment]
This is stale enough that I'm going to close this. It's been superseded by other projects like the remote dev boxes.

**Pull request closed without merging**